Makefile

```diff
@@ -15,7 +16,7 @@ help:
 .PHONY: help Makefile
 
 autobuild:
-	sphinx-autobuild -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)/html" --re-ignore "central-api|_build"
+	sphinx-autobuild -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)/html" --re-ignore "central-api|_build" --host 0.0.0.0
```

Comment:
This change allows running `make autobuild` in a container (with Docker for example) with port forwarding.